### PR TITLE
[Publisher][Uploaded Files] Remove "Mark as Pending" and "Mark as Processed" buttons

### LINKED ISSUE #1574: [Publisher][Uploaded Files] Remove "Mark as Pending" and "Mark as Processed" buttons
CSG requests that we remove these two buttons from the UI.

Reference area: https://github.com/Recidiviz/justice-counts/blob/7e05a749d595765a4d0443cc25e6519ee1f1467d/publisher/src/components/DataUpload/UploadedFiles.tsx#L210-L223

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

Removed "Mark as Pending" and "Mark as Processed" buttons

## Type of change

> All pull requests must have at least one of the following labels applied (otherwise the PR will fail):

| Label                       	| Description                                                                                               	|
|-----------------------------	|-----------------------------------------------------------------------------------------------------------	|
| Type: Bug                   	| non-breaking change that fixes an issue                                                                   	|
| Type: Feature               	| non-breaking change that adds functionality                                                               	|
| Type: Breaking Change       	| fix or feature that would cause existing functionality to not work as expected                            	|
| Type: Non-breaking refactor 	| change addresses some tech debt item or prepares for a later change, but does not change functionality    	|
| Type: Configuration Change  	| adjusts configuration to achieve some end related to functionality, development, performance, or security 	|
| Type: Dependency Upgrade      | upgrades a project dependency - these changes are not included in release notes                             |

## Related issues

closes #1574 

### Development

**This box MUST be checked by the submitter prior to merging**:
- [ ] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by mxosman** [comment]
Thank you, Ilya! Just a couple of small adjustment requests:

Can we have this cover the name of the sector just so it's more visually pleasing?
Screenshot 2024-11-12 at 11 12 00 AM

Something funny is going on with the styling of that Uploaded Files page - do you mind taking a look? It looks like there's some extra spacing on the left pushing everything over. I don't think it's related to this change, but just noticed it when I took a look and figured we can fix it here too.

Screenshot 2024-11-12 at 11 12 59 AM

File: publisher/src/components/DataUpload/UploadedFiles.tsx [hunk 1]
```diff
@@ -207,20 +207,6 @@ export const UploadedFileRow: React.FC<{
...
-                    />
-                  )}
-                  {(badgeText === "pending" || badgeText === "error") && (
-                    <Button
-                      label="Mark as Processed"
-                      onClick={() => updateUploadedFileStatus(id, "INGESTED")}
-                      labelColor="blue"
-                    />
-                  )}
                   {badgeText !== "error" && (
```
Comment (mxosman) [inline publisher/src/components/DataUpload/UploadedFiles.tsx:210]: Ah, sorry - can you get the "Mark as Error" as well? I'm not sure why I missed that. But basically all we want is the "Delete" and nothing else. So L208-218 can be removed

**Comment by mxosman** [comment]
Looks fantastic to me! Love the adjustment of keeping the Delete button after the sector name! Sorry, I forgot one more button that needs to be removed, then this is good to go!

File: publisher/src/components/DataUpload/UploadedFiles.tsx (hunk 1 above)
Comment (nasaownsky) [reply publisher/src/components/DataUpload/UploadedFiles.tsx]: Done :)

**Review by mxosman — APPROVED** [review]
THANK YOU!

**Pull request merged**